src/stores/assetsStore.ts

```diff
@@ -36,7 +39,9 @@ async function fetchInputFilesFromAPI(): Promise<AssetItem[]> {
  * Fetch input files from cloud service
  */
 async function fetchInputFilesFromCloud(): Promise<AssetItem[]> {
-  return await assetService.getAssetsByTag('input', false)
+  return await assetService.getAssetsByTag('input', false, {
+    limit: INPUT_LIMIT
```

Comment:
@viva-jinyi I don't think reducing the limit from 500 to 100 should impact your Assets work, but let me know if it does and we can adjust.